### DISCUSSION TITLE
Alias Shortcut for Fetching Languages

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#2982] [Feature Requests]
`?lang[translations]=en-US` or `?lang=en-US` instead of `?deep[translations][filter][languages_code]=en-US`

This is much easier to use/remember, and could support fallback language data... `?lang=en-US,nl-NL` (where `nl` is fallback).

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
